contents/handbook/engineering/feature-pricing.md

```diff
@@ -6,38 +6,61 @@ showTitle: true
 
 ## In an ideal world, Posthog’s pricing enables users and organizations to:
 
-- Use PostHog for free if they are hobbyists or pre-PMF
-- Experience the product before paying for it
-- Start paying when they are ready, on their own, with few hurdles
-- Transparently pay for the value they receive
-  - Eg. Usage-based pricing on events, recordings
-  - Eg. Paying per product so they only pay for what they use
-- Make it a no-brainer to pick PostHog over other competitors
+1. Use PostHog for free if they are hobbyists or pre-PMF.
+2. Experience the product before paying for it.
+3. Start paying when they are ready, on their own, with few hurdles.
+4. Transparently pay for the value they receive.
+  - e.g. Usage-based pricing on events, recordings.
```

Comment:
is this the proper way to do eg?? I've always seen it as `eg.` not `e.g.`

Reply:
It is, yeah, but it's one of those things that differs depending on the style guide. The Guardian (UK paper) doesn't use periods at all, AP style guide uses periods but also insists you use a comma after – `e.g., words here – which seems nuts to me!

TL;DR, it's mostly personal preference but e.g. is the most common usage.